app/routes/user.py

```diff
@@ -1,6 +1,28 @@
-"""
-This module provieds a router for the /user endpoint.
-"""
-from fastapi import APIRouter
+from fastapi import APIRouter, HTTPException
+from typing import List
+from app.models.pydantic.UserResponse import UserResponse  # Assurez-vous que ce chemin d'importation est correct
+from app.models.tortoise.user import UserInDB
+from app.utils.http_errors import CommonErrorMessages  # Chemin d'importation pour les messages d'erreur
 
-userRouter: APIRouter = APIRouter()
+# Création d'un routeur pour les utilisateurs
+userRouter = APIRouter()
+
+@userRouter.get("/{user_id}", response_model=UserResponse)
+async def get_user(user_id: int) -> UserResponse:
+    """
+    Récupère un utilisateur par son ID.
+    Si l'utilisateur n'est pas trouvé, une exception HTTP 404 est levée avec un message d'erreur.
+    """
+    user = await UserInDB.get_or_none(id=user_id)  # Tente de récupérer l'utilisateur dans la base de données
+    if user is None:  # Si l'utilisateur n'existe pas
+        raise HTTPException(status_code=404, detail=CommonErrorMessages.USER_NOT_FOUND)  # Renvoie une erreur 404
+    return user  # Renvoie les informations de l'utilisateur trouvé
+
+@userRouter.get("/", response_model=List[UserResponse])
+async def get_all_users() -> List[UserResponse]:
+    """
+    Récupère tous les utilisateurs de la base de données.
+    Renvoie une liste d'utilisateurs. Si aucun utilisateur n'est trouvé, renvoie une liste vide.
+    """
+    users = await UserInDB.all()  # Récupère tous les utilisateurs
```

Comment:
The /all types of routes should use a pagination module.
It will be provided soon.